src/main/java/nextstep/subway/path/ui/PathController.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.subway.path.ui;
+
+import nextstep.subway.path.application.PathService;
+import nextstep.subway.path.dto.PathResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/paths")
+public class PathController {
+    private PathService pathService;
```

Comment:
```suggestion
    private final PathService pathService;
```

컨벤션으로 이렇게 작성하는건 어떨까요?